src/components/core/tooltip/tooltip/tooltip-trigger.tsx

```diff
@@ -0,0 +1,25 @@
+import { ReactElement } from 'react';
+import * as TooltipPrimitive from '@radix-ui/react-tooltip';
+
+export type TTooltipTriggerProps = {
+    children: ReactElement;
+    setOpenTooltip?: (value: boolean) => void;
```

Comment:
`setOpenTooltip` sounds like it's setting a state. I think `onToggle` would be a good name for this function.